motioneye/uploadservices.py

```diff
@@ -29,6 +29,7 @@
 
 import boto3
 import pycurl
+import base64
```

Comment:
Since this is from the standard library, it needs to be moved into the upper block, for alphabetical sorting the very first entry. Also, I suggest to import only the used method. We currently don't really do this, but it makes sense in general to import only what is required, and we can start with this right here:
```suggestion
from base64 import b64encode
```